src/MTGCore.ClientApp/ClientApp/src/App.vue

```diff
@@ -44,138 +45,156 @@
               <div class="row">
                 <div class="col-lg-6">Cards in Deck</div>
                 <div class="col-lg-6">
-                  <DeckList ref="deckListRef"
-                            v-on:triggerChange="updateDeckCardList"
-                            v-on:addDeck="addNewDeck"
-                            v-bind:deckList="deckList"
-                            v-bind:selectedOption="selectedDeck"/>
+                  <DeckList
+                    ref="deckListRef"
+                    v-on:triggerChange="updateDeckCardList"
+                    v-on:addDeck="addNewDeck"
+                    v-bind:deckList="deckList"
+                    v-bind:selectedOption="selectedDeck"
+                  />
                 </div>
               </div>
             </div>
             <div class="card-body">
-              <DeckCardList v-bind:deckCards="deckCards"/>
+              <DeckCardList v-bind:deckCards="deckCards" />
             </div>
           </div>
         </div>
+        <SearchFilterModal v-on:submitFilter="SetSearchFilter" ref="SearchFilterRef"></SearchFilterModal>
```

Comment:
It might be worth putting the name search box in the same component as they're all filters? The search box itself doesn't have to go into the modal. The benefit of this is you can include the `Name` property as a filter in the filter object and emit it up with the rest of the filters, making things a bit cleaner.